<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="HW1916" name="Test-Login" tests="1" failures="1" timestamp="3 Oct 2017 08:58:55 GMT" time="37.676" errors="0">
  <testcase name="GS1_Login" time="4.026" classname="com.web.poc.script.GS1_Login">
    <failure type="java.lang.AssertionError" message="Test Case Failed because - Failed to Logout from GS1">
      <![CDATA[java.lang.AssertionError: Test Case Failed because - Failed to Logout from GS1
	at com.web.poc.script.GS1_Login.TestCaseStatus(GS1_Login.java:32)
	at com.web.poc.script.GS1_Login.GS1_Login(GS1_Login.java:84)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- GS1_Login -->
</testsuite> <!-- Test-Login -->
